lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -31,6 +41,10 @@ export default function LMSGrader({
   courseName,
   students: unorderedStudents,
 }) {
+  const {
+    api: { authToken, sync: syncAPICallInfo },
+  } = useContext(Config);
```

Comment:
It might make sense to extract this API call into a service in future, similar to how we do for VitalSource.